extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -40,6 +40,18 @@ function register_block() {
 					'type'    => 'boolean',
 					'default' => true,
 				),
+				'textColor'              => array(
+					'type' => 'string',
+				),
+				'customTextColor'        => array(
+					'type' => 'string',
+				),
+				'backgroundColor'        => array(
+					'type' => 'string',
+				),
+				'customBackgroundColor'  => array(
+					'type' => 'string',
+				),
```

Comment:
If the `custom` color comments earlier are relevant, then this would need work too. If this is all normal, then just dismiss my comment :)

Reply:
done!